sci-biology/bowtie/bowtie-1.1.2.ebuild

```diff
@@ -0,0 +1,58 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+inherit eutils toolchain-funcs
+
+DESCRIPTION="Popular short read aligner for Next-generation sequencing data"
+HOMEPAGE="http://bowtie-bio.sourceforge.net/"
+SRC_URI="mirror://sourceforge/bowtie-bio/${P}-src.zip"
+
+LICENSE="Artistic"
+SLOT="1"
+KEYWORDS="~amd64 ~x86 ~x64-macos"
+
+IUSE="examples tbb"
+
+DEPEND="app-arch/unzip
+	tbb? ( dev-cpp/tbb )"
+RDEPEND=""
+
+DOCS=( AUTHORS NEWS TUTORIAL doc/README )
+HTML_DOCS=( doc/{manual.html,style.css} )
+
+src_prepare() {
+	# Suppress useless -Wall pollution
+	sed -i 's/\-Wall/\-Wno-enum-compare/g' Makefile || die
```

Comment:
Stuff like:

In file included from SeqAn-1.1/seqan/basic.h:64:0,
                 from SeqAn-1.1/seqan/sequence.h:27,
                 from SeqAn-1.1/seqan/index.h:27,
                 from ebwt_build.cpp:5:
SeqAn-1.1/seqan/basic/basic_holder.h: In function ‘void seqan::setValue(seqan::Holder<TValue, seqan::Tristate2>&, TValue&)’:
SeqAn-1.1/seqan/basic/basic_holder.h:1001:59: warning: typedef ‘THolderType’ locally defined but not used [-Wunused-local-typedefs]
  typedef typename Value<Holder<TValue, Tristate2> >::Type THolderType;
                                                           ^
SeqAn-1.1/seqan/basic/basic_holder.h: In function ‘typename seqan::Reference<seqan::Holder<TValue, seqan::Tristate2> >::Type seqan::value(seqan::Holder<TValue, seqan::Tristate2>&)’:
SeqAn-1.1/seqan/basic/basic_holder.h:1027:59: warning: typedef ‘THolderType’ locally defined but not used [-Wunused-local-typedefs]
  typedef typename Value<Holder<TValue, Tristate2> >::Type THolderType;
                                                           ^
SeqAn-1.1/seqan/basic/basic_holder.h: In function ‘void seqan::assignValue(seqan::Holder<TValue, seqan::Tristate2>&, const TSource&)’:
SeqAn-1.1/seqan/basic/basic_holder.h:1051:59: warning: typedef ‘THostValue’ locally defined but not used [-Wunused-local-typedefs]
  typedef typename Value<Holder<TValue, Tristate2> >::Type THostValue;

I do not see any utility in this (given that most of it comes from the included SeqAn libraries).